### DISCUSSION TITLE
Can we use some parts of outline (ie, the editor) for other projects?

### DISCUSSION THREAD

**Discussion by mwufi** [outline/outline#3405] [Self Hosting] [answered]
Hi! I'm a bit confused about the LICENSE!

I was wondering, what is outline's pricing for things like, forking the code and running a version for another product?

I'm thinking of building a toy product with Markdown editing, so wondering if that would be fine!

THanks!

**Comment by tommoor** [comment, marked as answer]
The license applies to any forks, it cannot be used commercially. However there is a true open source version of the editor code here:

https://github.com/outline/rich-markdown-editor
